transparency-in-coverage/python/processors/helper_suggestions/state_management_flattener.py

```diff
@@ -0,0 +1,320 @@
+import csv
+import os
+from io import FileIO
+
+import ijson
+
+from processors.helper_suggestions.hash_dict import hashdict
+from processors.helpers import MRFOpen, Row
+from processors.schema import SCHEMA
+
+
+class StateManagedFlattener:
```

Comment:
This class is branched off yours at some previous point in time so it probably won't work if you fix yours. It's meant more as a reference to explain some ideas.